rn/Teacher/src/common/components/rich-text-editor/__tests__/RichTextToolbar.test.js

```diff
@@ -51,20 +49,6 @@ describe('RichTextToolbar', () => {
     ).toMatchSnapshot()
   })
 
-  it('renders icons in RTL', () => {
-    I18nManager.isRTL = true
-    expect(
-      renderer.create(
-        <RichTextToolbar
-          {...defaultProps}
-          redo={jest.fn()}
-          undo={jest.fn()}
-        />
-      ).toJSON()
-    ).toMatchSnapshot()
-    I18nManager.isRTL = false
-  })
-
```

Comment:
the `icon()` function uses bundled images with directional metadata, so they will get displayed properly, and don't need to be manually flipped.